### PR TITLE
edit tags

### PR DESCRIPTION
**Pull request by TheScreechingBagel**
edited a few and added an apps tag

### CODE REVIEW

**Comment by TheScreechingBagel** [comment]
idk if linking fmhy & uBO is too bold

File: src/tags.toml [hunk 1]
```diff
@@ -50,6 +50,9 @@ isContextEnabled = true
 content = """
 # Can I play this movie or TV show in another language?
 It's not possible to watch content in other languages on movie-web. We don't intend to add this feature soon, so you're most likely better off searching for a website that provides content in your desired language!
+Here is a list of non-english :pirate_flag: sites:
+<https://fmhy.pages.dev/non-english>
+Find your language in the sidebar and look under the Streaming section. [Use this adblocker!](<https://ublockrigin.com>)
```
Comment (binaryoverload) [inline src/tags.toml:55]:
```suggestion
Find your language in the sidebar and look under the Streaming section. [Use this adblocker!](<https://ublockorigin.com>)
```

File: src/tags.toml [hunk 2]
```diff
@@ -50,6 +50,9 @@ isContextEnabled = true
 content = """
 # Can I play this movie or TV show in another language?
 It's not possible to watch content in other languages on movie-web. We don't intend to add this feature soon, so you're most likely better off searching for a website that provides content in your desired language!
+Here is a list of non-english :pirate_flag: sites:
+<https://fmhy.pages.dev/non-english>
```
Comment (binaryoverload) [inline src/tags.toml:54]:
```suggestion
Here is a list of non-English sites: <https://fmhy.pages.dev/non-english>
```

**Review by binaryoverload — CHANGES_REQUESTED** [review]

**Comment by binaryoverload** [comment]
I don't think linking fmhy & uBO is bold, although I suggested removing the 🏴‍☠️ reference and corrected the uBlock URL

File: src/tags.toml
Comment (TheScreechingBagel) [inline src/tags.toml:1]: maybe a better link/message for devs?

**Review by binaryoverload — APPROVED** [review]

**Pull request merged**